projects/core/src/cart/bundle/occ/adapters/default-occ-bundle-config.ts

```diff
@@ -0,0 +1,13 @@
+import { OccConfig } from 'projects/core/src/occ/config/occ-config';
+
+export const defaultOccBundleConfig: OccConfig = {
+  backend: {
+    occ: {
+      endpoints: {
+        bundleStart: 'users/${userId}/carts/${cartId}/bundles?fields=DEFAULT',
+        bundleAllowedProductsSearch:
+          'users/${userId}/carts/${cartId}/entrygroups/${entryGroupId}/allowedProductsSearch',
```

Comment:
Shall we specify the specific `?fields=...` configuration for `allowedProductsSearch`?